docs/API.md

```diff
@@ -6,6 +6,6 @@ Percona Everest provides a set of APIs that enable you to access its features pr
 
 Percona Everest lets you visually interact with API resources representing all objects within Percona Everest. You can browse the API using the UI, accessible at the `/swagger endpoint` URL. 
 
-To access the API, click [Percona Everest API](https://docs.percona.com/everest-api-doc/).
+To access the API, click [Percona Everest API](https://docs.percona.com/everest-api-doc/){:target="_blank"}.
```

Comment:
This link worked yesterday but now opens the doc home page.

Reply:
The docs are not online. Will work once the docs are on percona.com